uptime_sensor.h has been removed in 2024.7.0

Compilation fails with 2024.7.0 as uptime_sensor.h is missing.
It has been replaced with uptime_seconds_sensor.h and uptime_timestamp_sensor.h.
This is the breaking change: [https://github.com/esphome/esphome/pull/7029](https://github.com/esphome/esphome/pull/7029)

Thanks a lot for your hard work!

does not work either

Compiling .pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/api/subscribe_state.cpp.o
Compiling .pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/api/user_services.cpp.o
Compiling .pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/binary_sensor/automation.cpp.o
Compiling .pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/binary_sensor/binary_sensor.cpp.o
Compiling .pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/binary_sensor/filter.cpp.o
Compiling .pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/button/button.cpp.o
Compiling .pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/captive_portal/captive_portal.cpp.o
Compiling .pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/climate/climate.cpp.o
Compiling .pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/climate/climate_mode.cpp.o
Compiling .pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/climate/climate_traits.cpp.o
Compiling .pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/cn105/climateControls.cpp.o
In file included from src/esphome.h:35,
                 from src/esphome/components/cn105/Globals.h:2,
                 from src/esphome/components/cn105/cn105.h:2,
                 from src/esphome/components/cn105/climateControls.cpp:1:
src/esphome/components/cn105/uptime_connection_sensor.h:2:10: fatal error: esphome/components/uptime/uptime_sensor.h: No such file or directory
    2 | #include "esphome/components/uptime/uptime_sensor.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/aa-mitsubishi-porchemesa/src/esphome/components/cn105/climateControls.cpp.o] Error 1
========================== [FAILED] Took 7.49 seconds ==========================

Hello, I know this issue looks as fixed, and thank you for that, but how do I upload the fix to the device? (Sorry, not too used to ESPHome yet). These are the messages I get when trying to update wirelessly to 2024.7.0. Thanks.

```
Compiling .pioenvs/mitsubishi-esp32-office/src/esphome/components/cn105/cycle_management.cpp.o
In file included from src/esphome.h:33,
                 from src/esphome/components/cn105/Globals.h:2,
                 from src/esphome/components/cn105/cn105.h:2,
                 from src/esphome/components/cn105/climateControls.cpp:1:
src/esphome/components/cn105/uptime_connection_sensor.h:2:10: fatal error: esphome/components/uptime/uptime_sensor.h: No such file or directory
 #include "esphome/components/uptime/uptime_sensor.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/mitsubishi-esp32-office/src/esphome/components/cn105/climateControls.cpp.o] Error 1
In file included from src/esphome.h:33,
                 from src/esphome/components/cn105/Globals.h:2,
                 from src/esphome/components/cn105/cn105.h:2,
                 from src/esphome/components/cn105/cycle_management.cpp:2:
src/esphome/components/cn105/uptime_connection_sensor.h:2:10: fatal error: esphome/components/uptime/uptime_sensor.h: No such file or directory
 #include "esphome/components/uptime/uptime_sensor.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from src/esphome.h:33,
                 from src/esphome/components/cn105/Globals.h:2,
                 from src/esphome/components/cn105/cn105.h:2,
                 from src/esphome/components/cn105/cn105.cpp:2:
src/esphome/components/cn105/uptime_connection_sensor.h:2:10: fatal error: esphome/components/uptime/uptime_sensor.h: No such file or directory
 #include "esphome/components/uptime/uptime_sensor.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/mitsubishi-esp32-office/src/esphome/components/cn105/cycle_management.cpp.o] Error 1
*** [.pioenvs/mitsubishi-esp32-office/src/esphome/components/cn105/cn105.cpp.o] Error 1
In file included from src/esphome.h:33,
                 from src/esphome/components/cn105/Globals.h:2,
                 from src/esphome/components/cn105/cn105.h:2,
                 from src/esphome/components/cn105/componentEntries.cpp:1:
src/esphome/components/cn105/uptime_connection_sensor.h:2:10: fatal error: esphome/components/uptime/uptime_sensor.h: No such file or directory
 #include "esphome/components/uptime/uptime_sensor.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/mitsubishi-esp32-office/src/esphome/components/cn105/componentEntries.cpp.o] Error 1
```

--- a/components/cn105/uptime_connection_sensor.cpp
+++ b/components/cn105/uptime_connection_sensor.cpp
@@ -9,7 +9,7 @@ namespace esphome {
         static const char* const TAG = "uptime.connection.sensor";
         void HpUpTimeConnectionSensor::update() {
             if (connected_) {
-                UptimeSensor::update();
+                UptimeSecondsSensor::update();
             } else {
                 this->uptime_ = 0;
                 this->publish_state(0);

--- a/components/cn105/uptime_connection_sensor.h
+++ b/components/cn105/uptime_connection_sensor.h
@@ -1,10 +1,10 @@
 #pragma once
-#include "esphome/components/uptime/uptime_sensor.h"
+#include "esphome/components/uptime/uptime_seconds_sensor.h"
 
 namespace esphome {
     namespace uptime {
 
-        class HpUpTimeConnectionSensor : public UptimeSensor {
+        class HpUpTimeConnectionSensor : public UptimeSecondsSensor {
         public:
             void update() override;
             std::string unique_id() override;

